docs/install/local/README.md

```diff
@@ -81,6 +75,7 @@ xcode-select --install
 
    For Windows:
     ```
+    mkdir cd c:\temp
```

Comment:
As discussed on the review, if `c:\temp` isn't the standard temporary file location on Windows, then there's no reason for these instructions to require its use.

I believe you said `cd %Temp%` would take you to a more appropriate temp location?

Reply:
In the review, we (I) said...

> Don’t have a temp directory on windows
> * Can use %temp% (decided to include mkdir c:\temp instead)
> * Can use a target location with `tar`, the next command
> * S: We should mkdir c:\temp
> * Steve to fix this ([PR 54](https://github.com/flowforge/installer/pull/54/))
> * ZJ: Extracting installer files to the right location takes more time than expected

This is a bit more nuanced than it first appears. Typically, a less experienced user would not know what `%temp%` is and if they managed to browse there, they would be fairly horrified at the crud they find in there (`%temp%` typically resolves to `c:\Users\<my-account>\AppData\Local\Temp` & on any given day contains a ton of weird crud) 

So it felt more aligned with Linux/MacOS to specify a fixed location like `c:\temp`

I can give this more thought if need be?

Also open to suggestions.